astropy/cosmology/connect.py

```diff
@@ -1,9 +1,16 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
```

Comment:
Wouldn't it be better to import `Any` or anything else that is only needed for annotations in `if TYPE_CHECKING:` blocks?

Reply:
Can do. Ruff has the rule set "TCH" which should check for this kind of thing. I agree this looks like it can be moved. Strange if it isn't picking it up.

Reply:
[By default imports from `typing` are excluded](https://docs.astral.sh/ruff/settings/#flake8-type-checking) from [TCH](https://docs.astral.sh/ruff/rules/#flake8-type-checking-tch) rules. We might want to change that setting.

Reply:
Yes! Do you want to push a PR? I can rebase.

Reply:
If we are changing the setting then it's best to do it quickly. I will open a pull request.